electron/main.ts

```diff
@@ -971,7 +1026,7 @@ ipcMain.handle('updateGame', async (e, game) => {
   }
 
   const epicOffline = await isEpicServiceOffline()
-  if (epicOffline) {
+  if (epicOffline && runner == 'legendary') {
```

Comment:
maybe the check for the epic status service can be done only if `runner === 'legendary'`? now it seems to trigger the check for the epic service even if not needed